code/modules/holomaps/holomap_datum.dm

```diff
@@ -0,0 +1,185 @@
+#define HOLOMAP_WALKABLE_TILE "#66666699"
+#define HOLOMAP_CONCRETE_TILE "#FFFFFFDD"
+
+/datum/holomap_interface
+	var/mob/activator = null
+	var/obj/item/holder = null
+	var/list/holomap_images = list()
+	var/image/holomap_base
+	var/static/image/default_holomap
+
+/datum/holomap_interface/New(obj/item/holder)
+	src.holder = holder
+	..()
+
+/datum/holomap_interface/Destroy()
+	QDEL_NULL(holder)
+	return ..()
+
+/datum/holomap_interface/process()
+	update_holomap()
+
+/datum/holomap_interface/proc/draw_special()
+	return
+
+/datum/holomap_interface/proc/draw_special_icon(var/filter, var/obj/holomap_holder)
```

Comment:
https://github.com/TauCetiStation/TauCetiClassic/wiki/Code-Convention#%D0%9E%D0%B1%D1%8A%D1%8F%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D0%B0%D1%80%D0%B3%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2-%D0%B2-%D0%BF%D1%80%D0%BE%D1%86%D0%B5%D0%B4%D1%83%D1%80%D0%B0%D1%85